karma.conf.js

```diff
@@ -27,49 +27,49 @@ module.exports = config => {
         served: true,
         included: false
       },
-      'test/APITests.js',
-      'test/EmptyTrackTests.js',
-      'test/ChromosomeLabelsTests.js',
-      'test/OverlayTrackTests.js',
-      'test/PngExportTest.js',
-      'test/ViewConfigEditorTests.js',
-      'test/ViewManipulationTests.js',
-      'test/ViewportProjectionTests.js',
-      'test/ZoomTests.js',
-      'test/GenbankFetcherTests.js',
-      'test/SchemaTests.js',
+      'test/AxisTests.js',
```

Comment:
Reordered these so that they are alphabetical.

Reply:
Nice